### DISCUSSION TITLE
Insights Timeseries chart where the value is the aggregation

### DISCUSSION THREAD

**Discussion by Joey92** [directus/directus#13081] [Archive]
Hi guys,

I'm trying to create a Timeseries chart on my Dashboard but the chart stays empty.

Specific usecase: I want to see the amount of users that were active in the last 24h in time buckets.

I've got the chart configured to count the users grouped by hour, but I still need to select a value field that is in my collection?

The count is basically the value it needs to graph.

**Comment by rijkvanzanten** [comment]
You should be able to use count on the primary key 🤔
